crypto/keys/pem_key_provider_test.go

```diff
@@ -0,0 +1,95 @@
+package keys
```

Comment:
Weird. I was concerned I might have caused a regression with #445 or #447, so I just deleted the license header on a file and ran presubmit locally, but it seems fine:

```bash
$ scripts/presubmit.sh ; echo "exit code = $?"
running golint
running go vet
running gocyclo
65 integration RunCTIntegrationForLog ./integration/ct_integration.go:67:1
running misspell
checking license header
Missing license header in: ./crypto/keys/pem_signer_factory_test.go
exit code = 5
```

Anyway, if you see anything strange happening (again) let me know.